packages/driver/src/cypress/runner.ts

```diff
@@ -1342,11 +1342,13 @@ export default {
     const replacePreviousAttemptWith = (test) => {
       const prevAttempt = _testsById[test.id]
 
-      const prevAttempts = prevAttempt.prevAttempts || []
+      const prevAttempts = prevAttempt?.prevAttempts || []
```

Comment:
out of curiousity, why did you need to touch this code?

Reply:
this code was throwing errors in the app component tests job for some reason - see the link to the failure in the PR description. This is what fixes those tests in the contributor workflow

Reply:
Weird. Are those component tests retrying? I'm just trying to figure out why that would happen because there should be a previous attempt here if the currentRetry is 1 or more. I'm more so concerned there is some time of regression here, but I have no idea why this would be specifically on contributor PRs. any idea? I'm also OK with patching this after we get builds working it isn't a must to figure out right now imo